ci-operator/step-registry/fips-check/fips-check-commands.sh

```diff
@@ -8,6 +8,10 @@ set -o pipefail
 get_nodes=$(oc --request-timeout=60s get nodes -o jsonpath --template '{range .items[*]}{.metadata.name}{","}{end}')
 
 IFS="," read -r -a nodes <<< "$get_nodes"
+TEST_NS="openshift-e2e-fips-check"
+
+oc create ns "$TEST_NS"
+oc label namespace "$TEST_NS" --overwrite pod-security.kubernetes.io/enforce=privileged
```

Comment:
Need to label pod-security.kubernetes.io/audit=privileged as well. Otherwise will trigger alert pod-security.kubernetes.io/enforce=privileged? Trigger alert is not good thing for any cluster.

Reply:
We are running with "restricted" enforcement by default now, audit/warns should follow the enforcement label, I think.

Reply:
Test result:
Without setting pod-security.kubernetes.io/audit=privileged, the initially un-fired alert "PodSecurityViolation" is fired after executing your PR's commands. Demo:
[xxia@pres 2022-09-01 22:50:06 CST my]$ TEST_NS="openshift-e2e-fips-check"
[xxia@pres 2022-09-01 22:51:13 CST my]$ oc create ns "$TEST_NS"
namespace/openshift-e2e-fips-check created
[xxia@pres 2022-09-01 22:51:37 CST my]$ oc label namespace "$TEST_NS" --overwrite pod-security.kubernetes.io/enforce=privileged
namespace/openshift-e2e-fips-check labeled
[xxia@pres 2022-09-01 22:54:00 CST my]$ oc --request-timeout=60s -n "$TEST_NS" debug node/$NODE -- cat /proc/sys/crypto/fips_enabled || true
Warning: would violate PodSecurity "restricted:latest": ...
...
Removing debug pod ...

And repeat this command a few times ...

Then found:
[xxia@pres 2022-09-01 22:57:00 CST my]$ PATTERN='violate(s|) PodSecurity'
[xxia@pres 2022-09-01 22:59:04 CST my]$ PODS=$(oc get po -n openshift-kube-apiserver -l apiserver --no-headers | grep -o "^[^ ]*")
[xxia@pres 2022-09-01 22:59:15 CST my]$ for i in $PODS
do
  oc rsh -n openshift-kube-apiserver -c kube-apiserver $i bash -c "grep -hE '$PATTERN' /var/log/kube-apiserver/audit*.log || true"
done | jq -cs 'sort_by(.requestReceivedTimestamp)' | jq -c '.[]' > audit-of-violate-podsecurity.log
[xxia@pres 2022-09-01 22:59:27 CST my]$ jq '.userAgent[0:24] + " " + .requestReceivedTimestamp + " " + .annotations."pod-security.kubernetes.io/audit-violations"[0:50]' audit-of-violate-podsecurity.log
"oc/4.11.0 (linux/amd64)  2022-09-01T14:54:13.627956Z would violate PodSecurity \"restricted:latest\": hos"
"oc/4.11.0 (linux/amd64)  2022-09-01T14:56:01.159347Z would violate PodSecurity \"restricted:latest\": hos"
"oc/4.11.0 (linux/amd64)  2022-09-01T14:56:49.210533Z would violate PodSecurity \"restricted:latest\": hos"
"oc/4.11.0 (linux/amd64)  2022-09-01T14:56:52.981544Z would violate PodSecurity \"restricted:latest\": hos"
"oc/4.11.0 (linux/amd64)  2022-09-01T14:56:56.384130Z would violate PodSecurity \"restricted:latest\": hos"
"oc/4.11.0 (linux/amd64)  2022-09-01T14:56:59.109279Z would violate PodSecurity \"restricted:latest\": hos"

In Console, checked the alert, it gets fired.

Reply:
> audit/warns should follow the enforcement label

I think audit/warns only follow "restricted" when "enforce" is "restricted". But in your PR, "enforce" is privileged.